spec/requests/users_controller_spec.rb

```diff
@@ -2,18 +2,19 @@
 
 describe UsersController do
   describe "#perform_account_activation" do
-    let(:user) { Fabricate(:user, active: false) }
-    let(:email_token) { Fabricate(:email_token, user: user) }
     let!(:channel) { Fabricate(:chat_channel, auto_join_users: true) }
 
     before do
       Jobs.run_immediately!
       UsersController.any_instance.stubs(:honeypot_or_challenge_fails?).returns(false)
       SiteSetting.send_welcome_message = false
-      SiteSetting.chat_enabled = true
     end
 
     it "triggers the auto-join process" do
+      user = Fabricate(:user, last_seen_at: 1.minute.ago, active: false)
+      email_token = Fabricate(:email_token, user: user)
+      SiteSetting.chat_enabled = true
```

Comment:
why is this moved here?

Reply:
Thought it was necessary, but it's not. Will revert.